versions/v1.1.0/values.yaml

```diff
@@ -34,6 +34,8 @@ koordlet:
     kubeletLibDir: /var/lib/kubelet/
     koordProxyRegisterDir: /etc/runtime/hookserver.d/
     koordletSockDir: /var/run/koordlet
+  service: false
```

Comment:
just use one option seems enough, since it is not necessary to only install Service itself.
 e.g. enableServiceMonitor